app/feature/feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/data/ChangeAddressRepository.kt

```diff
@@ -109,6 +109,7 @@ private fun MoveIntentCreateMutation.Data.MoveIntentCreate.MoveIntent.toMoveInte
   maxApartmentNumberCoInsured = maxApartmentNumberCoInsured,
   maxHouseNumberCoInsured = maxHouseNumberCoInsured,
   extraBuildingTypes = extraBuildingTypes.map { it.toExtraBuildingType() },
+  oldAddressCoverageDurationDays = this.currentHomeAddresses.first().oldAddressCoverageDurationDays,
```

Comment:
As of now, we don't even have an entry point to the flow if we have more than one home insurance/address, but as far as I understand, it may be something that will be added soon. So here we assume that there is always one, I think. And if there will be more than 1 - I guess we'll have to redo the logic anyway